jetty-gcloud/jetty-gcloud-session-manager/src/main/config-template/etc/sessions/gcloud/session-store.xml

```diff
@@ -10,11 +10,11 @@
   <Call name="addBean">
    <Arg>
     <New id="sessionDataStoreFactory" class="org.eclipse.jetty.gcloud.session.GCloudSessionDataStoreFactory">
-       <Set name="gracePeriodSec"><Property name="jetty.session.gracePeriod.seconds" default="3600" /></Set>
+       <Set name="gracePeriodSec" property="jetty.session.gracePeriod.seconds"/>
        <Set name="savePeriodSec" property="jetty.session.savePeriod.seconds"/>
        <Set name="maxRetries"><Property name="jetty.session.gcloud.maxRetries" default="5"/></Set>
-       <Set name="backoffMs" property="jetty.session.gcloud.backoffMs"/>
-       <Set name="namespace" property="jetty.session.gcloud.namespace"/>
+       <Set name="backoffMs"><Property name="jetty.session.gcloud.backoffMs" default="1000"/></Set>
```

Comment:
Can we change this default in the code?  I think changing numeric defaults is OK, as it doesn't change behaviour, just when the behaviour happens.

Reply:
Actually I did default it in the code, I just forgot to remove it from the xml. I'll do that.